test/test_coap_listen.py

```diff
@@ -90,71 +90,24 @@ def test_plugin_init(mocker):
     assert coap.aiocoap_ctx is None
 
     # WHEN
-    loop.call_soon(coap.plugin_start(config))
-    await asyncio.sleep(.5)  # required to allow ensure_future task to complete
+    coap.plugin_start(config)
+    await asyncio.sleep(.3)
 
     # THEN
     assert coap.aiocoap_ctx is not None
-    assert 1 == log_info.call_count
+    assert 2 == log_info.call_count
     calls = [
         call('CoAP listener started on port {} with uri {}'.format(config['port']['value'], config['uri']['value']))]
     log_info.assert_has_calls(calls, any_order=True)
```

Comment:
can we verify the another call as well? I see now we have made `assert 2 == log_info.call_count`

Reply:
done